action.yml

```diff
@@ -69,6 +74,6 @@ inputs:
       converted to HTTPS.
     default: false
 runs:
-  using: node16
+  using: node12

```

Comment:
why the node downgrade? i know we were using old node elsewhere do we have to use it here too?